tests/e2e/constants/DevfilesRegistry.ts

```diff
@@ -0,0 +1,19 @@
+/*********************************************************************
```

Comment:
More relevant file name is `DevfileRegistryConstants`, IMHO

Reply:
let it be )